python-sdk/tests/benchmark/datasets.md

```diff
@@ -140,95 +140,3 @@ The [script](synthetic_ndjson_generator.py) is used to generate the NDJSON file.
 | five_gb    | 4.5 GB | NDJSON   | s3://astro-sdk/benchmark/synthetic-dataset/ndjson/five_gb.ndjson    |
 | five_gb    | 5 GB   | NDJSON   | s3://astro-sdk/benchmark/synthetic-dataset/ndjson/five_gb/          |
 | ten_gb     | 10 GB  | NDJSON   | s3://astro-sdk/benchmark/synthetic-dataset/ndjson/ten_gb/           |
-
-## Origin of the real datasets
```

Comment:
This dataset was originally used as a realistic dataset. Shouldn't we still keep it?

Reply:
We are not adding those numbers obtained by these datasets in benchmarking results either. right?